ang/civicase/activity/actions/services/resume-draft-activity-action.service.js

```diff
@@ -0,0 +1,46 @@
+(function (angular) {
+  var module = angular.module('civicase');
+
+  module.service('ResumeDraftActivityAction', ResumeDraftActivityAction);
+
+  /**
+   * Resume Draft Action
+   *
+   * @param {object} $rootScope rootscope object
+   * @param {object} viewInPopup common factory to open an activity in a popup
+   * @param {object} ActivityType activity type service
+   */
+  function ResumeDraftActivityAction ($rootScope, viewInPopup, ActivityType) {
+    /**
+     * Check if the Action is enabled
+     *
+     * @param {object} $scope scope object
+     * @returns {boolean} if the action is enabled
+     */
+    this.isActionEnabled = function ($scope) {
+      var isBulkAction = $scope.mode === 'case-activity-bulk-action';
+
+      if (!isBulkAction) {
+        var activityTypeName = ActivityType.findById($scope.selectedActivities[0].activity_type_id).name;
```

Comment:
to avoid this line being too long just capture the activity type object, not the name directly:
```js
var activityType= ActivityType.findById($scope.selectedActivities[0].activity_type_id);
```
Then when using it you can use `activityType.name`, which is almost as long as `activityTypeName`.

Reply:
Fixed it like this
```
var activityTypeId = $scope.selectedActivities[0].activity_type_id;       
var activityTypeName = ActivityType.findById(activityTypeId).name;
```